docs/getting-started/custom-gateway.md

```
@@ -1,47 +1,130 @@
+    gateway:
+      mode: public
+    fcm:
+      credentials_json: |
+        {
+          "type": "service_account",
+          "project_id": "your-project"
+        }
+    ```
+    **Implementation Steps**:
+
+    1. Follow [Private Server Guide](./private-server.md)
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)